pkg/backend/openshiftcluster/create.go

```diff
@@ -203,8 +205,12 @@ func (m *Manager) Create(ctx context.Context) error {
 		return err
 	}
 
-	image := &releaseimage.Image{
-		PullSpec: "arosvc.azurecr.io/openshift-release-dev/ocp-release-nightly@sha256:5f1ff5e767acd58445532222c38e643069fdb9fdf0bb176ced48bc2eb1032f2a",
+	image := &releaseimage.Image{}
```

Comment:
Is the plan in the future to call private Cincinnati three?

Reply:
perhaps we will let the user choose within a range of versions that they can deploy